addons/fleet/models/fleet_vehicle_log_contract.py

```diff
@@ -70,6 +71,20 @@ def _compute_contract_name(self):
                 name = record.cost_subtype_id.name + ' ' + name
             record.name = name
 
+    @api.depends('vehicle_id')
+    def _compute_needs_renewal(self):
+        today = fields.Date.from_string(fields.Date.today())
```

Comment:
why not? :eyes:  I see it's other places like that.. but it does not make much sense :smile: 
```suggestion
        today = fields.Date.today()
```